### PR TITLE
Update additional instances of inline damage to new format

### PR DESCRIPTION
**Pull request by ammalagonc**

### CODE REVIEW

File: packs/pfs-season-4-bestiary/burrowing-viper-rat.json [hunk 1]
```diff
@@ -62,7 +62,7 @@
                 },
                 "category": "offensive",
                 "description": {
-                    "value": "<p><strong>Saving Throw</strong> @Check[fortitude|dc:16]</p>\n<p><strong>Maximum Duration</strong> 4 rounds</p>\n<hr />\n<p><strong>Stage 1</strong> [[/r (1d6)[poison]]] damage and @UUID[Compendium.pf2e.conditionitems.Item.Enfeebled]{Enfeebled 1} (1 round)</p>\n<p><strong>Stage 1</strong> [[/r (1d8)[poison]]] damage and @UUID[Compendium.pf2e.conditionitems.Item.Enfeebled]{Enfeebled 1} (1 round)</p>"
+                    "value": "<p><strong>Saving Throw</strong> @Check[fortitude|dc:16]</p>\n<p><strong>Maximum Duration</strong> 4 rounds</p><hr /><p><strong>Stage 1</strong> @Damage[(1d6)[poison]] damage and @UUID[Compendium.pf2e.conditionitems.Item.Enfeebled]{Enfeebled 1} (1 round)</p>\n<p><strong>Stage 1</strong> @Damage[(1d8)[poison]] damage and @UUID[Compendium.pf2e.conditionitems.Item.Enfeebled]{Enfeebled 1} (1 round)</p>"
```
Comment (simonward) [inline packs/pfs-season-4-bestiary/burrowing-viper-rat.json:65]:
```suggestion
                    "value": "<p><strong>Saving Throw</strong> @Check[fortitude|dc:16]</p>\n<p><strong>Maximum Duration</strong> 4 rounds</p><hr /><p><strong>Stage 1</strong> @Damage[1d6[poison]] damage and @UUID[Compendium.pf2e.conditionitems.Item.Enfeebled]{Enfeebled 1} (1 round)</p>\n<p><strong>Stage 2</strong> @Damage[1d8[poison]] damage and @UUID[Compendium.pf2e.conditionitems.Item.Enfeebled]{Enfeebled 1} (1 round)</p>"
```

**Comment by simonward** [comment]
For Pernicious Spore Bomb and Sulfur Bomb, the inline damage links can probably be removed as it should be handled by the damage rolls.

**Comment by ammalagonc** [comment]
Changes made - please check when you get a moment

**Comment by simonward** [comment]
> Changes made - please check when you get a moment

I think splash is automatically handled as well?

**Comment by ammalagonc** [comment]
> > Changes made - please check when you get a moment
> 
> I think splash is automatically handled as well (Pernicious Spore Bomb)?

Pernicious Spore Bomb has three types of damage - two are included in the item itself, except the last one which I left alone.

**Comment by simonward** [comment]
> Pernicious Spore Bomb has three types of damage - two are included in the item itself, except the last one which I left alone.

I think it's automatically handled by the:
```
"splashDamage": {
	"value": 1
}
```

Screen Shot 2024-08-24 at 7 22 46 PM

**Comment by ammalagonc** [comment]
Updated

File: packs/equipment/sulfur-bomb-greater.json [hunk 2]
```diff
@@ -21,7 +21,7 @@
...
+            "value": "<p><strong>Activate</strong> <span class=\"action-glyph\">1</span> Strike</p><hr /><p>A thick, sulfurous, irritating gas fills this golden-yellow flask. You gain a +2 item bonus to attack rolls. The bomb 3d4 acid damage and 3 acid splash damage. On a hit, the target takes a -1 status penalty to Perception checks and attack rolls until the end of its next turn, or becomes @UUID[Compendium.pf2e.conditionitems.Item.Sickened]{Sickened 1} on a critical hit. Creatures hit with this bomb are temporarily immune to the effects of the bomb for 1 minute.</p>\n<p>@UUID[Compendium.pf2e.equipment-effects.Item.Effect: Sulfur Bomb]</p>"
```
Comment (simonward) [inline packs/equipment/sulfur-bomb-greater.json:24]:
```suggestion
            "value": "<p><strong>Activate</strong> <span class=\"action-glyph\">1</span> Strike</p><hr /><p>A thick, sulfurous, irritating gas fills this golden-yellow flask. You gain a +2 item bonus to attack rolls. The bomb deals 3d4 acid damage and 3 acid splash damage. On a hit, the target takes a -1 status penalty to Perception checks and attack rolls until the end of its next turn, or becomes @UUID[Compendium.pf2e.conditionitems.Item.Sickened]{Sickened 1} on a critical hit. Creatures hit with this bomb are temporarily immune to the effects of the bomb for 1 minute.</p>\n<p>@UUID[Compendium.pf2e.equipment-effects.Item.Effect: Sulfur Bomb]</p>"
```

**Review by simonward — APPROVED** [review]

**Pull request merged**